app/src/main/java/org/dhis2/usescases/datasets/dataSetTable/dataSetSection/MapFieldValueToUser.kt

```diff
@@ -46,11 +47,6 @@ class MapFieldValueToUser(
                     field.value()
                 }
             }
-            ValueType.IMAGE,
-            ValueType.FILE_RESOURCE,
-            ValueType.TRACKER_ASSOCIATE,
-            ValueType.REFERENCE,
-            ValueType.GEOJSON -> resources.getString(R.string.unsupported_value_type)
```

Comment:
so, now when there is a field of type File resource for example... shouldn't we show "Unsupported value type"?

Reply:
I had the same question when reviewing, than I notice that this part was duplicated. @mmmateos remove the duplication.

Reply:
ok, thanks!